astropy/table/table.py

```diff
@@ -150,6 +150,39 @@ def keys(self):
     def values(self):
         return list(OrderedDict.values(self))
 
+    def isinstance(self, *cls):
```

Comment:
It is a minor thing, but by analogy with `isinstance` itself, I think I'd just put `cls` here (i.e., without the `*`), so that one has to put in an actual `tuple` if one wants to test for multiple classes. cc @taldcroft since it was his suggestion.

Also, are we sure to have this as a public method? (I think it is OK, just checking...)